lib/Mojolicious/Plugin/Yancy.pm

```diff
@@ -508,6 +510,15 @@ sub register {
     $formats->{ tel } = sub { 1 };
 }
 
+# if false or a ref, just returns same
+# if non-ref, treat as JSON-containing file, load and decode
+sub _ensure_json_data {
+    my ( $data ) = @_;
+    return $data if !$data or ref $data;
+    # assume a file in JSON format: load and parse it
+    decode_json path( ( $ENV{MOJO_HOME} || () ), $data )->slurp;
```

Comment:
You should probably pass-in `$app` so you can do `$app->home->child( $data )` here instead of checking the environment variable.

Reply:
Oo, great idea! Will do immediately.